legal-api/src/legal_api/reports/report.py

```diff
@@ -97,6 +106,8 @@ def _get_report(self):
         if response.status_code != HTTPStatus.OK:
             return jsonify(message=str(response.content)), response.status_code
 
+        self._document_service.create_document(self._business.identifier, self._filing.identifier, self._report_key, jwt.get_account_id(), response.content)
```

Comment:
Are we moving all documents to document service? I thought the plan is to move only the static documents (uploaded by users) which are in minio

Reply:
My understanding is we are moving all documents that get generated and have a filing id to the document service

Reply:
And I believe we have a separate ticket to replace minio with document service?